### ISSUE TITLE
could you please add some systemd auto run script?

### ISSUE THREAD

**Issue by youxiaojie** [wangyu-/UDPspeeder#312]
could you please add some systemd auto run script?
and/or add some deb/rpm relation files?
thanks!

**Comment by wangyu-** [comment]
no, I only implememt the core functions that is necessary to be done in c++

I don't maintain systemd scripts
